### PR TITLE
Reclamation KS7 Map Option + Balancing tweaks and bugfixes

### LINKED ISSUE #2658: [bug] Plasma Grenades are now instant killing people.
#### Description of issue

plasma grenades, upon sticking to someone, Instantly kill you. Tested with both an ODST and a Spartan, it killed both instantly. Straight to braindead with no chance of survival

#### Difference between expected and actual behavior

The Plasma nade shouldn't be an instant kill, this was changed

#### Steps to reproduce 
have funny covenant man stick you with a nade
have it blow up and instantly kill you

#### Specific information for locating

the plasma grenade code itself (possibly to do with what Blox changed about them) or it may be due to the nerf to brainmed(Done by Cael)

#### Length of time in which bug has been known to occur

1 day? I'd assume

#### Client version, Server revision & Game ID

Server Revision: 8f0c5215d7519d766659c9630f55c42a3c7b38c1 - alpha - 
Game ID: b75-doYS
Current map: Geminus City

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### LINKED ISSUE #2666: [BUG] LR Slipspace not ending round
#### Description of issue
Long range slipspace, when activated and charged, acts like a short range slipspace.  You are able to activate it and such, but the round indeed does not end.  Once it activates, you are able to go back into real space just like short range.

#### Difference between expected and actual behavior
Long range should end the round.

#### Steps to reproduce 
Something happens that requires you to activate long range
Activate long range
Wait to see that it does not actually end the round.

#### Specific information for locating

image
The standard text prompts and such, just no leaving.

#### Length of time in which bug has been known to occur

Unknown, but most likely recent.

#### Client version, Server revision & Game ID

Client Version: 513
Server Revision: b60cbd3e15f11e6567a76f3dab8d37ec08dff486 - alpha - 
Game ID: b77-az7P
Current map: Geminus City

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### LINKED ISSUE #2672: Huragok unable to be played
#### Description of issue
At the moment, huragok seems broken.  For one, there is no option to select it as a job.  Either a cyborg species or option would probably fix that.  I have not seen what other issues there are (because I haven’t played it)

#### Length of time in which bug has been known to occur

Unknown.
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### LINKED ISSUE #2684: Lowering the Gravity Stun Timer
#### Description of issue

If you are playing a race which does not have access to magboots (unggoy, for example), the stun that happens when you go from a non-gravity area to a gravity area is crippling. It's currently about 10 seconds from what I can tell, which is an eternity in SS13 time, while the enemy can just wail on you, making it a death sentence. For other people, if you go out of a window into space and then get pushed back into that window with your magboots off, you are instantly stunned. I suggest a decrease of this time to something a little more reasonable, or at the very least the ability to crawl.

### LINKED ISSUE #2685: Plasma Grenades
#### Description of issue

Playing the game over the recent weeks, one thing has stood out to me as inherently unbalanced, that being plasma grenades. 

The entire premise of cooking a grenade and throwing it is tossed out the window in favor of sticking, which with a 100% chance, ashes the limb it is attached to, and often equipment. For a marine, this is pretty much an instantly kill, with Spartans still dying pretty frequently from this. 

On the other hand, the UNSC grenade is very underwhelming. I have stood on frag grenades a few times and been fine, and to use them effectively you have to either be very lucky or be very good at timing, since grenades can be inconsistent in fuse timers. 

Getting stuck by a constantly respawning suicide grunt carrying 4 grenades isn't fun in the slightest, and while I'm sure I will get a list of reasons in which they are ineffective and actually need to be buffed, the reality is that most combat happens within 5-6 tiles, and in that range plasma grenades are the most lethal weapon in the game. 

So how do we change it? There are a few ways, all of which I am open to. The first way is eliminating the ashing effect grenades have. This is the simplest, but not the best, as it does make the grenade pretty weak, as the UNSC grenade has the ability to (in very unlikely circumstances) to gib a limb. 

The second way is to reduce stick chance. This was tried in the past, but the Covenant players complained that it made grenades nearly effective. While I disagree with this idea, the game is about having fun, and if they are not fun to play with, they are not fun to play with. 

The third and final way I will propose is making them have an arm time before they are able to stick, a couple seconds or something. This will give the UNSC player a warning that a grenade is about to fucking rail them, and that they should probably put on the jukes. If after that the Covenant are still able to stick them, more power to them. This is the best option in my opinion, because it makes it simulates the warning someone would have by a grenade being armed and thrown at them, rather than just being instantly stuck and dead before they can physically react. 

I'd love to hear alternatives, if you have any.

### LINKED ISSUE #2690: Poplock seems broken
#### Description of issue
Poplock seems to be counting ghosted players as active. This throws a monkey wrench into poplock mechanics, as if a player decides to ghost and just watch the round, they are taking up a poplock slot. 

#### Difference between expected and actual behavior
Ghosted players shouldn't be counted. 

#### Steps to reproduce 
1. Die
2. Ghost
3. Watch as no one can spawn on a particular side because it has more epic funni ghosts

#### Length of time in which bug has been known to occur

4 days

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [ x] Issue could be reproduced at least once
- [x ] Issue could be reproduced by different players
- [x ] Issue could be reproduced in multiple rounds
- [x ] Issue happened in a recent (less than 7 days ago) round
- [x ] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### LINKED ISSUE #2693: Remove Weapon Drop from Explosions
#### Description of issue

When the embassy vote went up about explosions being changed, quite literally 0 people voted for weapon drop to be added along with it. 

This thread will feature stats from UNSC weapons because I haven't bothered to do research into the covenant weapons, but you can look for yourselves if you Ctrl + F 

`can_use_when_prone()
	return 1`

While the explosion itself doesn't actually knock the weapon out of your hand, 90% of weapons cannot be held while prone. The total list of UNSC weapons that can are:

MA5B
DMR
SRS

I think one handed weapons can also be held on the ground, but I am not sure about that. 

Two of those are very seldom used, and very hard to use at close range where you are even at threat of getting hit by an explosive. This means that if you are hit with an explosive, you are most of the time instantly separated from your gun, where you cannot even shoot back at the person shooting you with an explosive. With weapons like the concussion rifle, fuel rod, or M41, this makes it basically a first draw is the winner for explosives, which is really just not fun. 

Ideally make it so weapons don't get knocked out of your hands when you get hit with an explosive. It makes gameplay very stun based, which just isn't fun. 

Likewise, if the knockback range of explosives could be reduced, that'd be ideal. We voted for a /slight/ knockback, not being sent 4 blocks down the road.

### LINKED ISSUE #2696: ODP Hitting Itself and Being Hit By AI Ships When Enemy Ship Is Too Close
#### Description of issue
When an enemy ship is too close, when the ODP fires to shoot it, it actually hits itself. The ODP will also shield the enemy ship from AI ship fire.

#### Difference between expected and actual behavior
ODP should not be able to hit itself. AI shouldn't be targeting friendlies.

#### Steps to reproduce 
Fly the CRS right on top of the ODP.

#### Length of time in which bug has been known to occur

Very recent.

#### Client version, Server revision & Game ID

ea21ec16334e470ba7cc463f37b5ec0be7c08084
b8f-dCgt

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### PR DESCRIPTION
**Pull request by X0-11**
should help with vehicles deleting all their occupants for fun and profit
:cl: XO-11
rscadd: mapvotes now contain a ks7 map variant for reclamation usage.
tweak: lowers rad amounts inflicted by some cov guns.
tweak: tweaks covenant precision weapons to act as more accurate counterparts to their unsc examples.
tweak:: gives plasma grenades a short adherence-activate delay so instant activate-and-throw plasnades are less common
tweak: Covenant weapon acquisition method has been modified as a pilot to see the effects of differing weapon restrictions based on initial player loadout choices.
tweak: Worhby.
/:cl:
closes #2696
closes #2693
closes #2690
closes  #2685
closes #2684
closes #2658
closes #2666 
closes #2672

### CODE REVIEW

File: code/modules/emotes/definitions/audible.dm [hunk 1]
```diff
@@ -182,13 +182,3 @@
 	playsound(user.loc, scream_sound,50,0,7)
 	h.next_scream_at = world.time + SCREAM_COOLDOWN
 	return
-
-/decl/emote/audible/species_sound/wort
-	key = "wort"
-	emote_message_3p = "USER worts, three times.!"
```
Comment (caelaislinn) [inline code/modules/emotes/definitions/audible.dm:188]: triggered

Comment (X0-11) [reply code/modules/emotes/definitions/audible.dm:188]: *wort

File: code/modules/halo/emotes/emotes.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+/decl/emote/audible/species_sound/wort
+	key = "wort"
+	emote_message_3p = "USER worts, three times.!"
```
Comment (caelaislinn) [inline code/modules/halo/emotes/emotes.dm]: STILL TRIGGERED

Comment (X0-11) [reply code/modules/halo/emotes/emotes.dm]: *wort2

**Review by caelaislinn — CHANGES_REQUESTED** [review]
why did you pull shit out of the armoury?

**Comment by X0-11** [comment]
they were placed into loadout boxes but the vendors are still there to hold minor nonrestricted stuff

File: code/modules/halo/emotes/emotes.dm (hunk 2 above)
Comment (caelaislinn) [reply code/modules/halo/emotes/emotes.dm]: `.!`

Comment (X0-11) [reply code/modules/halo/emotes/emotes.dm]: oh that's what you were triggered at

**Comment by X0-11** [comment]
errant punctuation fixed

**Comment by caelaislinn** [comment]
Where are the loadouts actually defined? I don't like the idea

**Comment by caelaislinn** [comment]
#2684 #2685

**Comment by X0-11** [comment]
I would rather provide greater weapon variety overall through loadouts whilst restricting OP weapon combos form forming than just placing the weapons in the vendors.

**Review by caelaislinn — DISMISSED** [review]
Can you remove the loadout stuff from this PR, then put it into a new PR by itself?

**Comment by X0-11** [comment]
Would rather not, it addresses a problem that the covenant has brought up multiple times recently and thus seems fairly critical to address.

**Comment by caelaislinn** [comment]
It's a significant feature, and a significant change to the existing system as well. I'd like to go through it without holding up the other stuff in this PR

**Comment by X0-11** [comment]
Multiple changes were made to the mapfile which have had further changes layered ontop of them. I'm unsure how to cleanly decouple the two without flat out redoing it all.

**Comment by caelaislinn** [comment]
What are the other changes you made to the CRS?

**Comment by X0-11** [comment]
various map modifications to add and move some stuff that's pod related iirc. Those are the changes I remember, but I'm not entirely sure.

File: code/modules/halo/vehicles/vehiclebase.dm [hunk 3]
```diff
@@ -353,6 +353,7 @@
 		anchored = 1
 	else
 		. = ..()
+	update_occupant_z_level(newloc.z)
```
Comment (caelaislinn) [inline code/modules/halo/vehicles/vehiclebase.dm]: This should only be necessary when moving up or down a zlevel, or when doing an overmap move

Comment (X0-11) [reply code/modules/halo/vehicles/vehiclebase.dm]: What about when traversing z-levels through bumptele?

Comment (caelaislinn) [reply code/modules/halo/vehicles/vehiclebase.dm]: Add something in the bumptele code to tell the vehicle to update their passenger zlevel

Comment (X0-11) [reply code/modules/halo/vehicles/vehiclebase.dm]: Will do.

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**